app/src/main/res/layout/fragment_privacy.xml

```diff
@@ -1,10 +1,9 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:card_view="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:layout_marginTop="@dimen/margin_moderate"
     android:layout_marginBottom="@dimen/margin_moderate"
+    android:background="#fff"
```

Comment:
Use background color from colors file. Don't hardcode colors value in the xml file